web-common/src/metrics/ErrorEventHandler.ts

```diff
@@ -37,12 +104,16 @@ export class ErrorEventHandler {
     ]);
   }
 
-  public fireHTTPErrorBoundaryEvent(
+  private fireHTTPErrorBoundaryEvent(
     api: string,
     status: string,
     message: string,
     screenName: MetricsEventScreenName,
   ) {
+    if (dev) {
```

Comment:
Nit:
1. The `metricsService` already keeps track of `dev` via the `is_dev` "common field". Is there a way to avoid this redundancy?
2. If not, `isDev` could be a property of the `ErrorEventHandler` class, rather than `dev` being pulled in here as a global.

Reply:
Dont want to pull information from `metricsService`. Better to send through `ErrorEventHandler` constructor.